homeassistant/components/deconz/sensor.py

```diff
@@ -149,7 +149,7 @@ class DeconzSensorDescription(SensorEntityDescription, DeconzSensorDescriptionMi
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
     ),
     DeconzSensorDescription[AirQuality](
-        key="pm2_5",
+        key="air_quality_pm2_5",
```

Comment:
shall I also update functions to match the new syntax? @Kane610

Reply:
I'm not following :lay

Reply:
nevermind, its ok like it is :)